package.json

```diff
@@ -17,10 +17,7 @@
     "build:frontend": "node scripts/copy-monaco-editor-files.js && npm run relay-compiler && vite build --base=./",
     "build:backend": "tsc --project server/tsconfig.json",
     "build": "npm run build:frontend && npm run build:backend",
-    "compile:armv7": "pkg . --out-path ./bin/ --targets node12.18.1-linux-armv7",
-    "compile:arm64": "pkg . --out-path ./bin/ --targets node12.18.1-linux-arm64",
-    "compile:x64": "pkg . --out-path ./bin/ --targets node12.18.1-win-x64,node12.18.1-macos-x64,node12.18.1-linux-x64",
-    "compile": "pkg . --out-path ./bin/ --target host",
+    "compile": "npx caxa -i . -o bin/dungeon-revealer-bin -- \"{{caxa}}/node_modules/.bin/node\" \"{{caxa}}/server-build/index.js\"",
```

Comment:
I feel that the output name should be something better than `dungeon-revealer-bin`